app/components/ErrorBoundary.tsx

```diff
@@ -53,10 +54,12 @@ export function ErrorBoundary() {
 					Something unexpected happened. Please try refreshing the page.
 				</p>
 				<div className="flex gap-2 justify-center">
-					<Button onClick={handleRefresh}>
-						<RefreshCw />
-						Retry
-					</Button>
+					<button onClick={handleRefresh} className="btn btn-primary">
+						<div className="flex gap-2 items-center">
```

Comment:
why are we adding this div wrapper here? We should avoid those things

Reply:
Done